e2e/openshift/openshift_packaging_test.go

```diff
@@ -109,9 +109,8 @@ func TestPackaging(t *testing.T) {
 	kubeconfig := os.Getenv("KUBECONFIG")
 	require.True(t, kubeconfig != "", "KUBECONFIG evn var not set")
 
-	// this call does some magic and ensures and sets
-	// oc in the PATH env variable for the test
-	paths.MaybeSetEnv("PATH", "oc", "hack", "bin", "oc")
+	// ensure oc will be on the path
```

Comment:
can you add some comments around how bazel is pulling oc down and how it is set up?  Should we have it documented how this code is interacting with the binaries and how bazel pulls down those binaries?